src/Common/Conjugation.fs

```diff
@@ -0,0 +1,32 @@
+module Conjugation
+
+open System
+open System.Collections.Generic
+
+open GrammarCategories
+
+type Pronoun =
+    | Já | Ty | To | My | Vy | Oni
+
+let getPronoun num pers =
+    match num, pers with
+    | Singular, First  -> Já
+    | Singular, Second -> Ty
+    | Singular, Third  -> To
+    | Plural,   First  -> My
+    | Plural,   Second -> Vy
+    | Plural,   Third  -> Oni
+
+let pronounToString pronoun =
+    match pronoun with
+    | Já  -> "já"
+    | Ty  -> "ty"
+    | To  -> "to"
```

Comment:
Either way I suggest having "ono" instead of "to".  Even though the latter is used far more often, grammar references usually have "ono" and it is also uniform with other pronouns. Users should understand the difference _on_, _ona_, _ono_ vs _ten_, _ta_, _to_ and mixing forms can confuse things a little.
```suggestion
    | To  -> "ono"
```